packages/react-components/src/atoms/Card.tsx

```diff
@@ -67,6 +68,7 @@ const Card: React.FC<CardProps> = ({
   accent = 'default',
   padding = true,
   stroke = false,
+  noShadow = false,
```

Comment:
```suggestion
  shadow = false,
```

Reply:
to match stroke. I hate all these hasShadow and noMargin etc just call it what it is and make it a boolean